boards/arm/nrf9160_pca10090/board.cmake

```diff
@@ -0,0 +1,4 @@
+board_runner_args(nrfjprog "--nrf-family=NRF91" "--softreset")

```

Comment:
Should be a comment explaining why we should have softreset on nRF91, but not on most other nrf platforms.

```
nrf52810_pca10040/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF52")

nrf52_pca10040/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF52")

nrf52840_pca10059/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF52")

nrf52840_pca10056/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF52")

nrf52_pca20020/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF52" "--softreset")

particle_xenon/board.cmake
8:board_runner_args(nrfjprog "--nrf-family=NRF52")

nrf51_ble400/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF51")

nrf9160_pca10090/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF91" "--softreset")

nrf52_adafruit_feather/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF52")

nrf51_pca10028/board.cmake
1:board_runner_args(nrfjprog "--nrf-family=NRF51")

particle_argon/board.cmake
7:board_runner_args(nrfjprog "--nrf-family=NRF52")

particle_boron/board.cmake
7:board_runner_args(nrfjprog "--nrf-family=NRF52")

```

Reply:
@SebastianBoe you are right on this. Actually, pin-reset seems to work on the latests kits, so we can remove this option. Will do.